app/assets/javascripts/media_player_wrapper/avalon_player_new.es6

```diff
@@ -294,7 +294,8 @@ class MEJSPlayer {
       success: this.handleSuccess.bind(this),
       embed_title: currentStreamInfo.embed_title,
       link_back_url: currentStreamInfo.link_back_url,
-      qualityText: 'Stream Quality'
+      qualityText: 'Stream Quality',
+      toggleCaptionsButtonWhenOnlyOne: 'true'
```

Comment:
Can we make `toggleCaptionsButtonWhenOnlyOne`'s value `boolean` rather than type `string`?   ie: `toggleCaptionsButtonWhenOnlyOne: true` instead of ``toggleCaptionsButtonWhenOnlyOne: 'true'`

Reply:
I can try that and see if it works.

Reply:
That worked.  I've updated this PR.